src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationProvider.cs

```diff
@@ -971,5 +974,18 @@ private IEnumerable<ConfigurationSetting> GetCurrentKeyValueCollection(string ke
 
             return currentKeyValues;
         }
+
+        private void EnsureTracing()
```

Comment:
Why do we give it such a generic name while it's all about feature management version detection?

Reply:
Ok that's true, I'll fix this to match the variable name you suggested as well